### ISSUE TITLE
Add drush sub-theming commands for Drupal 7.x version

### ISSUE THREAD

**Issue by AlexBrown-appnovation** [18F/web-design-standards-drupal-d7#4]
Any thoughts on throwing some drush features in the theme for some easy sub-theming? I figure it may be best to leave the USWDS codebase separate from any customized code so that it can be easily updated when new versions are released. This would avoid conflicts when updating and keep the codebase nice and clean at the same time.

**Comment by fureigh** [comment]
@AlexBrown-appnovation Thanks for opening this issue! I'd been thinking of keeping the USWDS codebase in an `assets` folder and [encouraging standard organization for subthemes](https://www.drupal.org/node/2165673)... can you elaborate on how you'd use drush features here?

**Comment by fureigh** [comment]
([Drupal 7 equivalent](https://www.drupal.org/node/225125) of the above, which was a Drupal 8-related link.)

**Comment by AlexBrown-appnovation** [comment]
Yes we would absolutely encourage the standard organization of sub themes while adding some handy drush commands that would allow users to very quickly create their sub-themes without all the copy/paste hassle.

If you look in the codebase of projects like Zen or AdaptiveTheme, they include this functionality. As I read and think about this however, perhaps this is more of a phase-2 item or 'nice-to-have' rather than a 'must-have'.

And yes, I agree with keeping everything in the assets folder as well!

**Comment by fureigh** [comment]
Ah, so this is specifically about drush commands for creating subthemes. Cool. I'll mark it `enhancement` for now.

**Comment by fureigh** [comment]
Took the liberty of changing the issue name; if you don't think that accurately reflects it, let me know.

**Comment by fureigh** [comment]
Once these are created, let's be sure to document them in #12.
